experiments/src/lib.rs

```diff
@@ -44,3 +40,67 @@ impl Experiments {
         &self.experiments
     }
 }
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+pub struct Experiment {
+    pub id: String,
+    pub filter_expression: String,
+    pub targeting: Option<String>,
+    pub enabled: bool,
+    pub arguments: ExperimentArguments,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ExperimentArguments {
+    pub slug: String,
+    pub user_facing_name: String,
+    pub user_facing_description: String,
+    pub active: bool,
+    pub is_enrollment_paused: bool,
+    pub bucket_config: BucketConfig,
+    pub features: Vec<String>,
+    pub branches: Vec<Branch>,
+    pub start_date: Option<DateTime<Utc>>,
+    pub end_date: Option<DateTime<Utc>>,
+    pub proposed_duration: u64,
+    pub proposed_enrollment: u64,
+    pub reference_branch: Option<String>,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+pub struct Branch {
+    pub slug: String,
+    pub ratio: u32,
+    pub group: Option<Vec<Group>>,
+    pub value: BranchValue,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum Group {
+    Cfr,
+    AboutWelcome,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct BranchValue {}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct BucketConfig {
+    pub randomization_unit: RandomizationUnit,
+    pub namespace: String,
+    pub start: u32,
+    pub count: u32,
+    pub total: u32,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "snake_case")]
```

Comment:
According to [the experiment schema](https://github.com/mozilla/nimbus-shared/blob/8bbb142b6cfdebc285b29d24e37fadc8bef0ee4c/types/experiments.ts#L60), it looks like `userId` isn't snake case.  I'm not sure if this is a bug in the schema or by design, but if we snake case here it probably wouldn't match the schema as it is defined right now.

Reply:
Wow that's a good catch! Thanks @travis79!!

Reply:
FYI: I asked in #nimbus-project on Slack to get more info on this